Nodejs/MigrateToJsps/Jsps/JspsProjectCreator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MigrateToJsps
+{
+    internal class JspsProjectCreator
+    {
+        private string njsprojDir;
+        private NjsprojFileModel njsprojFileModel;
+        private string newProjectParentDir;
+
+        public JspsProjectCreator(string njsprojDir, NjsprojFileModel njsprojFileModel, string newProjectParentDir)
+        {
+            this.njsprojDir = njsprojDir;
+            this.njsprojFileModel = njsprojFileModel;
+            this.newProjectParentDir = newProjectParentDir;
+        }
+
+        public void CreateJspsProject() // to be called by outsiders
+        {
+            var parentDir = Path.Combine(newProjectParentDir, njsprojFileModel.ProjectName + "-migrated");
```

Comment:
The other way around: NTVS gets "-old", jsps gets the old name :)

Reply:
done!